script/inject-sequences.js

```diff
@@ -6,11 +6,19 @@ sequences.sort((a, b) => b.length - a.length);
 
 const trie = new Trie();
 trie.addAll(sequences);
-const sequencePattern = trie.toString();
-console.log(sequencePattern);
 
-for (const file of ['index.js', 'text.js']) {
+const emojiSequence = trie.toString();
+console.log(emojiSequence);
+
+const emojiSequenceUnicode = trie.toString('u');
+console.log(emojiSequenceUnicode);
+
+for (const file of ['index.js', 'text.js', 'es2015/index.js', 'es2015/text.js']) {
```

Comment:
Can we tweak this section of the build script rather than duplicating the `src` files? WDYT?